qcut/resources/default-skills/qcut-toolkit/jianying-audio-reference/scripts/cache_probe.py

```
@@ -0,0 +1,264 @@
+    digest = hashlib.md5()
+    with file_path.open("rb") as input_file:
+        for chunk in iter(lambda: input_file.read(1024 * 1024), b""):
+            digest.update(chunk)
+    return digest.hexdigest()
+
+
+def probe_audio(file_path: Path) -> dict[str, object] | None:
+    ffprobe = shutil.which("ffprobe")
+    if not ffprobe:
+        return None
+    result = subprocess.run(
```

Ruff (0.16.0)

[error] `subprocess` call: check for execution of untrusted input

(S603)